server/controllers/notificationController.js

```diff
@@ -51,33 +51,60 @@ const notificationController = {
       })
       .then((notify) => {
         if (!notify) {
-          return res.redirect('/api/v1/notifications');
+          return res.status(404).json({
+            status: 'error',
+            error: 'Notification does not exist'
+          });
         }
         notify.isRead = true;
         notify.save()
           .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the server'));
-        const route = {
```

Comment:
why the deletion?

Reply:
I deleted it because the response redirects to that route and the redirection was blocking axios request to that endpoint. Instead of redirecting it returns a response.

Reply:
Alright